feature-libs/asm/assets/translations/en/asm.ts

```diff
@@ -71,6 +71,7 @@ export const customer360 = {
     },
     header: 'Product Reviews',
     emptyDescription: 'There are currently no product review items',
+    sku: 'SKU:',
```

Comment:
We probably want the colon outside the translation string.
```suggestion
    sku: 'SKU',
```